### PR TITLE
feat: spruce up options window

### PR DESCRIPTION
**Pull request by samsiegart**
image

### CODE REVIEW

File: source/options.html [hunk 1]
```diff
@@ -1,15 +1,23 @@
...
 <form id="options-form" class="detail-view-container">
-	<h3>Trusted Agoric Power Box Interfaces</h3>
-  <p>WARNING: only add pages here that you want to modify your personal
-  names!</p>
-  <ul id="agoric-powerbox-urls">
-    <li id="new-powerbox-url"><input type="text" placeholder="Trusted Power Box URL"></li>
-  </ul>
+  <h3>Trusted Agoric Power Boxes</h3>
+  <p>
+    WARNING: only add pages here that you want to modify your personal names!
```
Comment (michaelfig) [inline source/options.html]:
```suggestion
    WARNING: only add URLs here that you trust with your personal settings.
```

File: source/options.html [hunk 2]
```diff
@@ -1,15 +1,23 @@
...
+  <div class="new-powerbox-url" id="new-powerbox-url">
+    <mwc-textfield
+      type="url"
+      outlined
+      placeholder="Trusted Power Box Url"
+      spellcheck="false"
+    ></mwc-textfield>
+    <mwc-button raised label="Add"></mwc-button>
+  </div>
+  <h4>Receive dapp connections at...</h4>
```
Comment (michaelfig) [inline source/options.html]:
```suggestion
  <h4>Receive connections at...</h4>
```

File: package.json [hunk 3]
```diff
@@ -33,8 +33,15 @@
 		"extends": "stylelint-config-standard"
 	},
 	"dependencies": {
+		"@material/mwc-button": "^0.25.3",
```
Comment (michaelfig) [inline package.json]: Actually, could you please move most of these (everything except `patch-package`) to `devDependencies`?  They're not strictly necessary except for development.

**Review by michaelfig — APPROVED** [review]
Just a few minor things... overall, this is terrific and clarifying!  Great job!

File: package.json (hunk 3 above)
Comment (samsiegart) [reply package.json]: Done

File: source/options.html (hunk 2 above)
Comment (samsiegart) [reply source/options.html]: Fixed

File: source/options.html (hunk 1 above)
Comment (samsiegart) [reply source/options.html]: Fixed

**Pull request merged**